openllm-python/src/openllm/models/auto/factory.py

```diff
@@ -48,7 +50,12 @@ def for_model(cls,
     >>> llm = openllm.AutoLLM.for_model("flan-t5")
     ```
     '''
-    llm = cls.infer_class_from_name(model).from_pretrained(model_id=model_id, model_version=model_version, llm_config=llm_config, **attrs)
+    llm = cls.infer_class_from_name(model).from_pretrained(model_id=model_id,
+                                                           model_version=model_version,
+                                                           prompt_template=prompt_template,
+                                                           system_message=system_message,
```

Comment:
Why do you need to do this, You should already handle this in LLM class right?

Reply:
I believe this `for_model` function calls `from_pretrained`, which requires these parameters. `for_model` is called when I run this line in the sdk:
`llm_runner = openllm.Runner(model, llm_config=llm_config, prompt_template='{instruction}')`

Reply:
but from_pretrained should handle all attributes in `**attrs` 😄, and thats the whole point of kwargs and args

Reply:
Ah gotcha! I will make the change